examples/udt/udt.c

```diff
@@ -29,6 +29,7 @@
 #include <string.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <unistd.h>
```

Comment:
Needs to be updated for Windows; maybe something like this:

```diff
diff --git a/examples/udt/udt.c b/examples/udt/udt.c
index 21e63bde..3d9c9fed 100644
--- a/examples/udt/udt.c
+++ b/examples/udt/udt.c
@@ -29,7 +29,13 @@
 #include <string.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <unistd.h>
+#ifndef _WIN32
+# include <unistd.h>
+# define SLEEP(x) sleep(x)
+#else
+# include <windows.h>
+# define SLEEP(x) Sleep(x * 1000) // milliseconds
+#endif

 #include "cassandra.h"

@@ -247,7 +253,7 @@ void execute_schema_query(CassSession* session, const char* query) {

   version = schema_meta_version(session);
   while (version == existing_version && count < 20) {
-    sleep(1);
+    SLEEP(1);
     count++;
     version = schema_meta_version(session);
   }

```

Reply:
Thanks, simplified my solution. I think my original fix was a bit too paranoid.